client/src/components/modals/IncorrectGrantModal.js

```diff
@@ -0,0 +1,71 @@
+import api from 'api'
+import { Box, Button, Heading, Text, TextArea } from 'grommet'
+import * as React from 'react'
+import { Modal } from '../Modal'
+
+export const IncorrectGrantModalContent = ({ setShowing, setAlert }) => {
+  const [message, setMessage] = React.useState('')
+  const onChange = (newMessage) => {
+    setMessage(newMessage)
+  }
+  const onClick = () => {
+    setShowing(false)
+    if (message) {
+      api.user.submitGrantComplaint(message)
+      setAlert(
+        'Your message was sent to the ALSF Grants Team. They will be in touch with you shortly.'
+      )
+    }
+  }
+  return (
+    <Box
+      width="600px"
+      height="350px"
+      margin={{ left: 'large', right: 'large', bottom: 'large' }}
+      gap="medium"
+    >
+      <Box
+        fill="horizontal"
+        border={[{ size: 'small', side: 'bottom', color: 'black-tint-95' }]}
+        height={{ min: '50px' }}
+      >
+        <Heading serif margin={{ top: 'none', bottom: 'small' }} level="5">
+          Report Incorrect/Missing Information
+        </Heading>
+      </Box>
+      <Box direction="row" gap="small">
+        <Text weight="bold">Tell us what is wrong</Text>
+        <Text italic="true" color="red">
+          (Required)
+        </Text>
+      </Box>
+      <Box>
+        <Text size="small" color="black-tint-40">
+          We will notify the ALSF Grants Team in order to update your grant
+          information.
+        </Text>
+      </Box>
+      <Box height="250px">
+        <TextArea
+          height="200px"
+          fill
+          resize={false}
+          onChange={(event) => onChange(event.target.value)}
+        />
+      </Box>
+      <Box alignSelf="end" width="100px" margin={{ top: 'large' }}>
+        <Button label="Send" onClick={onClick} primary />
```

Comment:
could you disable the button when there is no message

Reply:
Yep! Done.